src/interfaces/index.ts

```diff
@@ -42,16 +42,54 @@ export type ICoin = {
   amount: string;
 };
 
+enum AuthenticatorTransport {
+	USB = "usb",
+	NFC = "nfc",
+	BLE = "ble",
+	Hybrid = "hybrid",
+	Internal = "internal",
+}
+
+interface CredentialFlags {
+	UserPresent: boolean;
+	UserVerified: boolean;
+	BackupEligible: boolean;
+	BackupState: boolean;
+}
+
+enum AuthenticatorAttachment {
+	Platform = "platform",
+	CrossPlatform = "cross-platform"
+}
+
+interface Authenticator {
+	AAGUID: string;
+	SignCount: number;
+	CloneWarning: boolean;
+	Attachment: AuthenticatorAttachment
+}
+
+export type ICredential = {
```

Comment:
Where is this used?

Reply:
Might need be used anywhere. @Peartes would know more